src/h5web/vis-packs/core/visualizations.ts

```diff
@@ -32,6 +35,7 @@ export enum Vis {
   Line = 'Line',
   Heatmap = 'Heatmap',
   Complex = 'Complex',
+  ComplexLine = 'Complex Line',
```

Comment:
I would remove the space here, to convey that the string is not used as a label.